lib/n_.js

```diff
@@ -18,9 +18,11 @@ var repl = require('repl'),
 // create new pristine `lodash` instance
 var _ = require('lodash').runInContext(server.context);
 
+var lodashToInject = process.argv.indexOf('--fp') === -1 ? _ : require('lodash/fp').runInContext();
```

Comment:
`runInContext` is used to avoid potential conflicts caused by augmenting the lodash package directly (i.e. extending it via `_.mixin`). The default context is `root` (`global`). By using `server.context` as the context object we make sure that we always access our pristine lodash inside the REPL and not the potentially extended `_` in the global scope. @jdalton Did I get it right?